parser.go

```diff
@@ -36,6 +36,7 @@ func (c *CommentList) Capture(values []string) error {
 // Node is the the interface implemented by all AST nodes.
 type Node interface {
 	Position() Position
+	EndPosition() Position
```

Comment:
`Comment` is the only place where EndPosition() is used, so it doesn't seem like it's necessary for it to be part of the base interface?

Reply:
Yeah, fair enough. I removed that and limited it only to `Comment`.